format/FormatDate.js

```diff
@@ -15,7 +15,8 @@ import {fmtSpan} from './FormatMisc';
 export const DATE_FMT = 'YYYY-MM-DD',
     DATETIME_FMT = 'YYYY-MM-DD h:mma',
     TIME_FMT = 'h:mma',
-    MONTH_DAY_FMT = 'MMM D';
+    MONTH_DAY_FMT = 'MMM D',
+    CALENDAR_DATE_FMT = 'YYYYMMDD';
```

Comment:
Not sure about this format?  Aren't these formats primary for display -- wouldn't we stick with YYYY-MM-DD

Reply:
I just used 'YYYYMMDD' as that was the format requested in the ticket - happy to use either format

Reply:
Tom -- think I meant here in the ticket that this would be the format for transferring it and storing it in the control, not displaying it!

(seemed like the tighter usage was our standard for API serialization)